_data/tools.yml

```diff
@@ -334,16 +334,62 @@
   v2: false
   v3: true
 
-# Parsers / Validators
+# Schema Validators
 
 - name: Speccy
-  category: parsers
+  category: schema-validators
   link: https://speccy.io/
   github: https://github.com/wework/speccy
   language: CLI
   description: Lint to enforce quality rules on your OpenAPI specifications
   v3: true
 
+- name: BigstickCarpet/swagger-cli
+  category: schema-validators
+  github: https://github.com/BigstickCarpet/swagger-cli
+  language: Node.js / CLI
+  description: A handy cli tool for validating JSON or YAML files schema files that respects $ref
+  v2: true
+  v3: true
+
+- name: openapi-spec-validator
+  category: schema-validators
+  github: https://github.com/p1c2u/openapi-spec-validator
+  language: Python
+  description: OpenAPI aSpec validator
+  v3: true
+
+- name: Dredd
+  category:
+    - schema-validators
```

Comment:
Can we put dredd into just testing? I think it runs a Schema validator before running its main logic but so do most tools, do we can’t really say it’s goal is to be a Schema validator. 

It is actually kinda a Data validator though I suppose. It makes requests then confirms the data matches what the Schema says. Let’s put it in there! :)

Reply:
Done